src/main/java/com/example/NiRi/controller/CartItemController.java

```diff
@@ -19,23 +15,34 @@ public class CartItemController {
     @Autowired
     private CartItemService cartItemService;
 
-    @GetMapping("/{user_id}")
-    public List<CartItem> getCartItemsByUserId(@PathVariable Long user_id) {
-        return cartItemService.getCartItemsByUserId(user_id);
+    @GetMapping("/{userId}")
+    public List<CartItem> getCartItemsByUserId(@PathVariable Long userId) {
+        return cartItemService.getCartItemsByUserId(userId);
     }
 
-    @PostMapping("/{userId}/add")
-    public void addToCart(@PathVariable Long userId, @RequestBody CartItemPayload cartItemPayload) {
-        cartItemService.addToCart(userId, cartItemPayload);
+    @PutMapping("/{userId}/add")
+    public ResponseEntity<String> addToCart(@PathVariable Long userId, @RequestBody CartItemPayload cartItemPayload) {
+        try {
+            CartItemRequest cartItemRequest = new CartItemRequest(cartItemPayload.getProductId(), cartItemPayload.getQuantity());
+            cartItemService.addToCart(userId, cartItemRequest);
+            return ResponseEntity.ok("Item added to cart successfully.");
+        } catch (Exception e) {
+            return ResponseEntity.status(HttpStatus.INTERNAL_SERVER_ERROR).body("Error adding item to cart.");
```

Comment:
same here, it's better to respond with a JSON instead of String

Reply:
Changed to JSON